scripts/prepare-commit.js

```diff
@@ -20,7 +22,7 @@ const script = async () => {
   console.log('====================');
   console.log();
 
-  const currentVersion = require('../lerna.json').version;
+  const currentVersion = loadJSON('../lerna.json').version;
```

Comment:
Maybe you can use dynamic import, like `await import('../lerna.json')`. If it works, it would be easier to read.